src/modules/agents/ui/components/agent-form.tsx

```
@@ -40,7 +40,7 @@ export const AgentForm = ({
     trpc.agents.create.mutationOptions({
       onSuccess: () => {
         onSuccess: () => {
```

Biome (1.9.4)

[error] Unexpected label.

Only loops should be labeled.
The use of labels for other statements is suspicious and unfamiliar.

(lint/suspicious/noConfusingLabels)